test/common.c

```diff
@@ -71,8 +72,18 @@ void test_common(void)
     TEST_MSG("Test dir '%s' isn't a directory\n", path);
     goto done;
   }
+
+  if (!TEST_CHECK((setlocale(LC_ALL, "en_US.UTF-8") != NULL) ||
+                  (setlocale(LC_ALL, "C.UTF-8") != NULL)))
```

Comment:
Do we need `en_US` or can we just use `C`? Is there a use case there the former is available and the latter is not?

Reply:
> Do we need `en_US`

Hmm... probably not.
I've dropped it (in a separate commit in case we need to revert it).